src/containers/flows/CancelBid/CancelBid.tsx

```diff
@@ -26,6 +26,7 @@ export const CancelBid = ({
 	onClose,
 }: CancelBidContainerProps) => {
 	const { bid, bidData, refetch, isLoading } = useBidData(domainId, auctionId);
+	console.log(auctionId);
```

Comment:
nit: I believe it was for debugging purpose

Reply:
ah yes thanks good spot!